app/templates/accessibility-statement.hbs

```diff
@@ -0,0 +1,44 @@
+{{page-title "Toegankelijkheidsverklaring"}}
+
+<LayoutDetail>
+  <div class="au-o-region-large">
+    <div class="au-o-layout">
+      <AuContent class="au-u-3-4@medium">
+        <AuHeading>Toegankelijkheidsverklaring</AuHeading>
+        <p>Agentschap Binnenlands Bestuur streeft ernaar zijn websites en mobiele applicaties toegankelijk te maken, overeenkomstig het <AuLinkExternal href="http://www.ejustice.just.fgov.be/eli/decreet/2018/12/07/2018032457/justel">Bestuursdecreet van 7 december 2018</AuLinkExternal>.</p>
+        <p>Deze toegankelijkheidsverklaring is van toepassing op <AuLinkExternal href="https://...">....vlaanderen.be</AuLinkExternal></p>
+
+        <h2>Nalevingsstatus</h2>
+        <p>Deze web applicatie voldoet gedeeltelijk aan de <AuLinkExternal href="https://www.w3.org/Translations/WCAG21-nl/">Richtlijnen voor Toegankelijkheid van Webcontent (WCAG) 2.1 Niveau AA</AuLinkExternal>, omdat niet aan de onderstaande eis(en) is voldaan.</p>
+
+        <h2>Niet-toegankelijke inhoud</h2>
+
+        <p>De onderstaande inhoud is niet-toegankelijk om de volgende reden(en):</p>
+        <ul>
+          <li>
+            <strong>Datum selectie:</strong> het datum selectie veld bevat een datum popup die niet toegankelijk is via het toetsenbord. (<AuLinkExternal href="https://www.w3.org/Translations/WCAG21-nl/#toetsenbord">Succescriterium 2.1.1 Toetsenbord</AuLinkExternal>). <br>
+            <em>Alternatief:</em> een datum kan via tekst ingegeven worden.
+            <br>
+          </li>
+        </ul>
+
+        <h2>Opstelling van deze toegankelijkheidsverklaring </h2>
+        <p>Deze toegankelijkheidsverklaring is opgesteld op 18/07/2023.</p>
+        <p>Voor deze toegankelijkheidsverklaring werd de toegankelijkheid van de vermelde web applicatie beoordeeld door het Agentschap Binnenlands Bestuur.</p>
+
+        <h2>Feedback en contactgegevens</h2>
+        <p>Voor alle vragen en opmerkingen over de toegankelijkheid van de vermelde websites van het Agentschap Binnenlands Bestuur kunt u contact opnemen via <a href="mailto:binnenland@vlaanderen.be" class="c-link">binnenland@vlaanderen.be</a></p>
+
+        <h2>Handhavingsprocedure</h2>
+        <p>Bij onbevredigende antwoorden op uw vragen en opmerkingen kunt u de Vlaamse Ombudsdienst contacteren via onderstaande gegevens.</p>
+        <p>
+          <strong>Vlaamse Ombudsdienst</strong><br>
+          Leuvenseweg 86 <br>
+          1000 Brussel <br>
+          <a href="mailto:info@vlaamseombudsdienst.be" class="c-link">info@vlaamseombudsdienst.be</a> <br>
+          Telefoon (gratis): <a href="tel:info@vlaamseombudsdienst.be" class="c-link">1700</a> <br>
```

Comment:
@Dietr Shouldn't it be the phone number instead of the email address in the `href`?

Reply:
Yes, that's a mistake. Created a PR with a fix: https://github.com/lblod/frontend-burgernabije-besluitendatabank/pull/75